examples/notebooks/rag/requirements.txt

```diff
@@ -0,0 +1,57 @@
+# Milvus
+pymilvus
+pymilvus[model]
+
+# datasets
+datasets
+
+## Torch and enbeddings
+torch
+sentence-transformers
+
+## --- Parquet
+pandas
+pyarrow
+
+## DPK 
```

Comment:
DPK dependencies should be already installed when we do  `pip install data-prep-toolkit-transforms-ray==0.2.1.dev1`